scrapi/base/__init__.py

```diff
@@ -169,9 +159,8 @@ def get_properties(self, result, property_list):
 
         properties = {}
         for item in property_list:
```

Comment:
Would you mind commenting this block, just so it's more apparent why you are doing what it is you are doing?